### ISSUE TITLE
bnd-maven-plugin exits with error: "Illegal repetition near index 1"

### ISSUE THREAD

**Issue by lefou** [bndtools/bnd#951]
I'm using the bnd-maven-plugin version 2.4.1 on Linux. I've defined a Maven property `namespace` and want to reference it in the `Export-Package` header like this: 

```
Import-Pakage: \
  scala*;version="[${scalaBinVersion},${scalaBinVersion}.50)", \
  *

Export-Package: \
  ${namespace}.*
```

When running Maven, it gives the following error message: 

```
[ERROR] Failed to execute goal biz.aQute.bnd:bnd-maven-plugin:2.4.1:bnd-process (default) on project domino_2.10: bnd error: Illegal repetition near index 1
[ERROR] \${namespace}(\..*)?
[ERROR] ^
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

What makes me wonder: Using the property `scalaBinVersion` in the `Import-Package` header above works as expected.

Here is my `mvn -version` output:

```
Apache Maven 3.3.3 (7994120775791599e205a5524ec3e0dfe41d4a06; 2015-04-22T13:57:37+02:00)
Maven home: /usr/share/maven-bin-3.3
Java version: 1.8.0_45, vendor: Oracle Corporation
Java home: /opt/oracle-jdk-bin-1.8.0.45/jre
Default locale: de_DE, platform encoding: UTF-8
OS name: "linux", version: "3.18.11-gentoo", arch: "amd64", family: "unix"
```

**Comment by lefou** [comment]
Oh, actually, also the `scalaBinVersion` varible is not expanded at all. It just gives another error, an warning to be correct:

```
[INFO] --- bnd-maven-plugin:2.4.1:bnd-process (default) @ domino_2.10 ---
[WARNING] No translation found for macro: maven.scalaBinVersion, in 
[WARNING] No translation found for macro: maven.scalaBinVersion, in -----------------
Warnings
000: No translation found for macro: maven.scalaBinVersion, in 

```

So, how am I supposed to reference any settings or property from the pom? Is there somewhere proper documentation about the bnd-maven-plugin?

The following line from the `README.txt` of this project is suggesting that variable expansion works as expected (e.g. as in maven-bundle-plugin):

```
    -includeresource: ${project.build.outputDirectory} # i.e target/classes
```

**Comment by lefou** [comment]
This looks a lot like #924.

When we can expect a new release of bnd-maven-plugin? What's holding you back?

**Comment by njbartlett** [comment]
It will be released as part of bnd 3.0

**Comment by bjhargrave** [comment]
Closing since #924 is already in master for the 3.0 release this summer.
